content/20-exploring-the-cluster/22-rancher-applications/index.en.md

```diff
@@ -30,18 +30,18 @@ Here are all the relevant details for the **`Longhorn`** chart, like the Current
 
 ### Step 3 (Longhorn):
 
-Before we install Longhorn, we have to make sure we have the prerequistes installed on each node in the cluster. Longhorn requires **`nfs`** and **`iscsi`**. Longhorn provides **`kubectl`** deployment manifests to install each package.
+Before we install Longhorn, we have to make sure we have the prerequistes installed on each node in the cluster. Longhorn requires **`nfs`** and **`iscsi`** under the hood. Longhorn provides deployment scripts via **`kubectl`** to install the dependencies.
 
-To install the dependencies, run these commnads using the kubectl shell provided by the Rancher Manager. On your keyboard, click the **`backtick`** key.
+To install the dependencies, we are going to run these commnads using the kubectl shell provided by the Rancher Manager. On your keyboard, click the **`backtick`** key.
```

Comment:
i prefer to keep instructions to the user in present imperative rather than future tense. recommend not making this change so it's clear the user needs to take action (and the workshop creator/automation is not going to do it for them).